addon-test-support/@ember/test-helpers/dom/click.ts

```diff
@@ -69,6 +70,8 @@ export function __click__(element: Element | Document, options: MouseEventInit):
   click('button', { shiftKey: true });
 */
 export default function click(target: Target, options: MouseEventInit = {}): Promise<void> {
+  const mergedOptions = assign({ buttons: 1, button: 0 }, options);
```

Comment:
Can the `buttons` and `button` values have variable representations to improve readability?

```js
const PRIMARY_BUTTON = 1;
const MAIN_BUTTON_PRESSED = 0;

//...

const mergedOptions = assign({ buttons: PRIMARY_BUTTON, button: MAIN_BUTTON_PRESSED }, options);
```

Reply:
Added a constants file to handle this.